swagger.yaml

```diff
@@ -31,6 +31,10 @@ paths:
           description: "Successfully returned a cache time for a given id"
           schema:
             $ref: "#/definitions/CacheTime"
+        404:
+          description: "No cachetime was found using the id provided"
+        500:
```

Comment:
Do we actually need the 500 responses listed here? I would have thought that they're implicit. If we start listing them, then they need to be in every endpoint (this one has only been added to the `/health` endpoint), and also what about other 5xx responses, like a Bad Gateway? I personally think we don't need them as they're a bit redundant, but I'm happy to be corrected. Maybe someone from ONS can clarify.

Reply:
Understood, we followed the swagger documentation in the dp-dataset-api and it was in there. Perhaps we leave this in here and ONS can clarify like you suggest.